recipe/build.sh

```diff
@@ -33,6 +33,7 @@ bash configure --prefix=$PREFIX \
                --with-geos=$PREFIX/bin/geos-config \
                --with-geotiff=$PREFIX \
                --with-hdf4=$PREFIX \
+               --with-cfitsio=$PREFIX \
```

Comment:
Can you add a test for it?

Reply:
> Can you add a test for it?

Just to clarify: In `run_test.py` from line 11.

Reply:
Should be done.  Had a typo "FTIS" vs "FITS" but I expect to pass.  Let me know what else you need